src/www/firewall_rules.php

```diff
@@ -367,8 +376,8 @@
                       <th class="hidden-xs hidden-sm"><?=gettext("Port");?></th>
                       <th class="hidden-xs hidden-sm"><?=gettext("Destination");?></th>
                       <th class="hidden-xs hidden-sm"><?=gettext("Port");?></th>
-                      <th class="hidden-xs hidden-sm"><?=gettext("Gateway");?></th>
-                      <th class="hidden-xs hidden-sm"><?=gettext("Schedule");?></th>
+                      <th class="hidden"><?=gettext("Gateway");?></th>
```

Comment:
I don't think we should hide gateway (and schedule) columns without them being visible on the overview page completely, policy based routing mistakes tend to cause quite some issues in peoples setups.

Reply:
Just trying to serve the 80% here.
A single column wich can be used to indicate 'special' configuration options like Gateway, Schedule, statetype, .... maybe?